src/components/OrganizationDashboard/Cards/CommonProgress/CommonProgress.tsx

```diff
@@ -18,39 +18,33 @@ export default function CommonProgress({
 	return (
 		<Grid container>
 			<Grid item md={5}>
-				<Box m={1}>
+				<Box m={1} mt={0}>
 					<Typography variant="subtitle2" noWrap>
 						{title}
 					</Typography>
 				</Box>
 			</Grid>
-			<Grid item md={7}>
-				<Grid container>
-					<Grid item md={9}>
-						<Box mt={1}>
-							<BorderLinearProgress
-								variant="determinate"
-								value={percentage}
-								color={color}
-							/>
-						</Box>
-					</Grid>
-					<Grid item md={3}>
-						<Typography variant="caption">{`${percentage} %`}</Typography>
-					</Grid>
-					<Grid item md={size === "xs" ? 4 : size === "md" ? 5 : 7}></Grid>
-					<Grid item md={size === "xs" ? 8 : size === "md" ? 7 : 5}>
-						<Box display="flex">
-							{/* <Box mt="1" mr="1">
-								<UpdateIcon color="disabled" fontSize="small" />
-							</Box> */}
-							<Typography
-								variant="caption"
-								color="textSecondary"
-								noWrap
-							>{`${getLastUpdatedInWords(new Date(date))}`}</Typography>
-						</Box>
-					</Grid>
+			<Grid item md={7} container>
```

Comment:
it is better to provide xs instead of md .to avoid unexpected behavior on small screens

Reply:
We are using md on application from beginning

Reply:
we should change that to xs as these props like xs, md, lg bubbles up. 
so if you have provided xs only it will have the same effect as md or lg

Reply:
ohky